.gitignore

```diff
@@ -1,2 +1,3 @@
 xip
 xip.linux
+.idea/
```

Comment:
Please remove this from the PR, you can globally ignore `.idea` by using the global .gitignore in your home directory.